Civi/Api4/EntityFinancialAccount.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ */
+
+namespace Civi\Api4;
+
+/**
+ * EntityFinancialAccount. Joins financial accounts to financial types.
+ *
+ * @see https://docs.civicrm.org/dev/en/latest/financial/financialentities/#financial-accounts
+ *
+ * @bridge entity_id financial_account_id
+ *
```

Comment:
@eileenmcnaughton if you want the AccountRelationship field automatically shown during join (like relationship type id is shown for related contacts or activity contact type is shown for activities) do this:
```suggestion
 *  @ui_join_filters account_relationship
```

Reply:
@colemanw I've pushed that up - although note the whole thing doesn't work atm